localstack/services/apigateway/provider.py

```diff
@@ -248,7 +254,8 @@ def put_rest_api(self, context: RequestContext, request: PutRestApiRequest) -> R
         response = rest_api.to_dict()
         remove_empty_attributes_from_rest_api(response)
         store = get_apigateway_store(account_id=context.account_id, region=context.region)
-        store.rest_apis[request["restApiId"]].rest_api = response
+        rest_api_container = store.rest_apis[request["restApiId"]]
+        rest_api_container.rest_api = response
```

Comment:
The "response" was already there, but I was scratching my head at this, we could rename response to rest_api_response? something more domain-specific.

Reply:
Oh yeah, sorry. This could be removed, I suppose I was using the rest_api_container at some point but removed it, so the "split" stayed. I'll remove.